commands/azureCommands/acr-build-logs-utils/stylesheet.css

```diff
@@ -0,0 +1,176 @@
+.accordion {
```

Comment:
>accordion [](start = 1, length = 9)

if you are interested, you may ask Sajay to see if he can arrange an UI review with UX team.